Docs/user_guide/modules/webserver.md

```diff
@@ -2,46 +2,83 @@
+- Start server: Launches web server listening on port `2016`. If the default port is in use, other ports are checked sequentially until an open port is found, allowing more than one Slicer web server to run concurrently.
+- Stop server: Stop web server.
+- Open static page in external browser: Display `docroot` using default operating system web browser found using `qt.QDesktopServices`.
+- Open static page in internal browser: Display `docroot` using Slicer built-in web browser instantiated using `slicer.qSlicerWebWidget()`.
+- Log output: If `Log to GUI` is enabled, access log and execution results are logged. Logs are cleared periodically.
+- Clear Log: Clear log output displayed in the module panel.
+- Advanced:
+  - Slicer API: Enable/disable use of [Slicer endpoints](#slicer-endpoints) associated with the `/slicer` path.
+  - Slicer API exec: Enable/disable remote execution of python code through `/slicer/exec` endpoint. See [Remote Control][#remote-control].
+  - DICOMweb API: Enable/disable support of [DICOMWeb endpoints](#dicomweb-endpoints) associated with the `/dicom` path.
+  - Static pages: Enable/disable serving of static files found in the `docroot`  associated with the `/` path.
+  - Log to Console: Enable/disable the logging of messages in the console.
+  - Log to GUI: Enable/disable the logging of messages in the module panel.
 
-Buttons in the module can launch either a desktop browser tab or a `qSlicerWebWidget` pointing to the server's demo page.
+## Static endpoints
 
-The demo page shows most of the features of the Slicer web api and is hosted using the Static api.
+Hosts files out of the module's `docroot` like any standard http server.
 
-## Remote Control
+Currently this is used just for examples.
 
-The web server can also be accessed via other commands such as `curl`. A `dict` can be returned as a json object by setting it in the `__execResult` variable.
+## Slicer endpoints
 
-For example, these command can be used to download data and change the screen layout:
+### Remote Control
+
+The web server can also be accessed via other commands such as `curl`. A `dict` can be returned as a json object by setting it in the `__execResult` variable and enabling the `Slicer API exec` feature in the `Advanced` section.
+
+For example, these commands may be used to download the MRHead sample data, change the screen layout and return a dictionary including the ID of the loaded volume:
 
 ```
-curl -X POST localhost:2016/slicer/exec --data "import SampleData; volumeNode = SampleData.SampleDataLogic().downloadMRHead(); slicer.app.layoutManager().setLayout(slicer.vtkMRMLLayoutNode.SlicerLayoutOneUpRedSliceView); __execResult = {'volumeNodeID': volumeNode.GetID()}"
+curl -X POST localhost:2016/slicer/exec --data "\
```

Comment:
This would not work on Windows. Line continuation character is `^` in .bat files and backtick in PowerShell, and I don't think it is practically feasible in the standard old Windows console.

I would add examples that can be copy-pasted as is, on any OS, even if they are a bit harder to read.